.github/workflows/release-automation.yaml

```diff
@@ -0,0 +1,31 @@
+name: Release Finch latest version
+on:
+  workflow_dispatch:
+  create:
+    tags:
+      - 'v*'
```

Comment:
This might run on all branches when a new tag is created, it's okay for now. Maybe adding a TODO to indicate this would help.

Reply:
Will test and update comments